actionview/lib/action_view/helpers/form_helper.rb

```diff
@@ -1114,6 +1114,11 @@ def fields(scope = nil, model: nil, **options, &block)
       #     raw('Accept <a href="/terms">Terms</a>.')
       #   end
       #   # => <label for="post_terms">Accept <a href="/terms">Terms</a>.</label>
+      #
+      #   label(:post, :terms) do |translation|
+      #     raw('Accept <a href="/terms">#{translation}</a>.')
```

Comment:
```suggestion
      #     raw("Accept <a href=\"/terms\">#{translation}</a>.")
```
Double quotes for interpolation